include/circt/Dialect/Moore/MooreOps.td

```diff
@@ -72,9 +86,37 @@ def InstanceOp : MooreOp<"instance", [
   }];
 
   let arguments = (ins StrAttr:$instanceName,
-                       FlatSymbolRefAttr:$moduleName);
-  let assemblyFormat = [{
-    $instanceName $moduleName attr-dict
+                       FlatSymbolRefAttr:$moduleName,
+                       Variadic<AnyType>:$inputs,
+                       Variadic<AnyType>:$outputs,
```

Comment:
Do we need to split inputs and outputs? It looks in Moore we pass even output values to an instance op so I feel it's easier to merge(and mix) them?

Reply:
Because the Moore InstanceOp does not have results. Distinguish the inputs and outputs will be beneficial to the next lowering procedure. Unless we add DirectionAttr into components of the op